AlaqmaMaryamZadibaKokweh/MALZ002.xml

```diff
@@ -0,0 +1,162 @@
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>                          
+                                           <material key="parchment"/>
+                                       </support>
+                                       <extent>
+                                           <measure unit="leaf"></measure>                                            
+                                           <dimensions type="outer">
+                                               <height unit="mm">340</height>
+                                               <width unit="mm">260</width>
+                                               <depth unit="mm">100</depth>
+                                           </dimensions>
+                                       </extent>
+                            <foliation>Unfoliated.</foliation>
+                            <condition key="good">Some traces of use, some minor damages.</condition>
+                         </supportDesc>
+                     </objectDesc>
+                       
+                       <decoDesc>
+                                                  
+                           <decoNote type="headpiece" xml:id="d1">
+                               <locus target="#3r" cert="low"/>
+                               <desc>Elaborate coloured ornamental band (vegetal motifs)</desc>
+                           </decoNote>   
+                           
+                           <decoNote type="drawing" xml:id="d2">                            
+                               <desc>A small protective design on the penultimate page (face)</desc>
+                           </decoNote>   
+                           
+                           <decoNote type="drawing" xml:id="d3">
+                               <desc>Crude recent drawing of an angel, in modern pen</desc>
+                           </decoNote>   
+                           
+                                                     
+                       </decoDesc>
+                       
+                       <additions>
+                           <list>
+                               <item xml:id="a1">                                
+                                   <desc type="DonationNote">The note mentions the donor and commissioner Tasfā Māryām and his relatives</desc>
+                                   <q xml:lang="gez">ዝመጽኀፍ፡ ዘወሀባ፡ ተስፋ፡ ማርያም፡ አቡሁ፡ ገብረ፡ ዋህድ፡ እሙ፡ ወለተ። 
+                                       ኪዳን፡ ዘአጽሐፋ፡ በንዋዩ፡ </q>
+                               </item>
+                               
+                               <item xml:id="a2">                                
+                                   <desc type="DonationNote">A lengthy donation note on the penultimate folium, 
+                                       mentioning in part the same persons as Additional note 1, written in red, 
```

Comment:
the same persons as `<ref type="item" target="#a1">Additional note 1</a1>`,

Reply:
ok